java-checks/src/test/files/checks/PrintfMisuseCheck.java

```diff
@@ -224,6 +232,9 @@ void foo(Calendar c){
     java.util.logging.Logger logger4 = getLog();
     logger4.log(java.util.logging.Level.WARNING, "som.foo.errorcode", 404); // Noncompliant
     this.loggerField.log(java.util.logging.Level.WARNING, "som.foo.errorcode", 404);
+    String additionalMessage = "...";
+    logger4.log(java.util.logging.Level.WARNING, "message " + additionalMessage, new Exception());  // Noncompliant {{Format specifiers should be used instead of string concatenation.}}
```

Comment:
this is a FP to me. You don't have the choice to use concatenation if you want to use the "Throwable" method.